UnityProject/Assets/Scripts/Tilemaps/Utils/TileList.cs

```diff
@@ -64,16 +64,18 @@ public bool HasObjects(Vector3Int position)
 	}
 	public List<RegisterTile> Get(Vector3Int position)
 	{
-		return _objects.ContainsKey(position) ? _objects[position] : emptyList;
+		return _objects.TryGetValue(position, out var objectsOut) ? objectsOut : emptyList;
 	}
 
-	public IEnumerable<RegisterTile> Get(Vector3Int position, ObjectType type) {
-		if ( !HasObjects( position ) )
+	public IEnumerable<RegisterTile> Get(Vector3Int position, ObjectType type)
+	{
+		if (_objects.TryGetValue(position, out var objectsOut) == false)
```

Comment:
```suggestion
		if (_objects.TryGetValue(position, out _) == false)
```
If you're not using the out value, you can dispose it like this.

Reply:
It is using the out value

Reply:
ah yeah, got confused by the return brackets.